config/config.go

```diff
@@ -10,7 +10,8 @@ import (
 
 // app meta
```

Comment:
warning: comment on exported const AppName should be of the form "AppName ..." (golint)